charts/cluster-api-runtime-extensions-nutanix/templates/csi/nutanix/manifests/helm-addon-installation.yaml

```diff
@@ -24,4 +24,7 @@ data:
       - effect: NoSchedule
         key: node-role.kubernetes.io/control-plane
         operator: Exists
+
+    # TODO Remove this once the Nutanix 3.0 CSI driver is GA.
+    supportedPCVersions: fraser-2024.1-stable-pc-0.1
```

Comment:
Do we have the same PC version on sherlock CI also ? Not sure where we are running our CI today.

Reply:
My manual test was run against sherlock.

Reply:
Confirmed same version.

Reply:
If it's temporary fix, should we add `fraser-2024.1-stable-pc-0` and `master` versions too?

Reply:
@manoj-nutanix this can take multiple versions then? Comma separated I guess?

Reply:
yes, something like -
```
 supportedPCVersions: "master,fraser-2024.1-stable-pc-0,fraser-2023.4-stable-pc-0,fraser-2024.1-stable-pc-0.1" 
```

I think, I tried this in past and it works but do test it one more time for confirmation.

Reply:
Tested locally and this works! Thanks @manoj-nutanix!